Terasology/WorkstationCrafting
# Added InGameHelp + WorkstationHelp + Bugfix

### Contains

Added InGameHelp, WorkstationHelp, ManualLabor, and Durability modules as dependencies. Also, added the appropriate new components in the item prefabs to utilize the new dependencies. Plus, added an optimization for loading recipes, as well as a bugfix to the upgrade recipes. Moreover, renamed the processes and stations to follow a standard naming convention.
### How to test

Attempt to craft items, view the items' descriptions on the help menu (press P), and upgrade the woodcrafting station from basic to novice to standard. Do the same for the stonecrafting station. Make sure all of the wood and stone items are craftable.
### Issues
- MapleTrunks become PineLogs when spawned using giveItem.
- NPE when placing Resin into the tool slot of a Workstation Crafting Station. That is, "WorkstationInventoryUtils.java:39".
### Credits

@Cervator for advice, and @Josharias for JoshariasSurvival which motivated me to add these features.
